.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
Cool! So we can later add more commands for lining, running tests, etc.

Reply:
Good idea! Let me think of filing an RFC for that.